src/InOneWeekend/hittable_list.h

```diff
@@ -18,6 +18,8 @@
 #include <memory>
 #include <vector>
 
+using std::shared_ptr;
+using std::make_shared;
```

Comment:
Odd; these should not be necessary — they're part of `rtweekend.h`. If they're not present at this point, they should be added to `rtweekend.h`.

Reply:
Oh yuck. `rtweekend.h` doesn't exist yet. Also, after introducing `rtweekend.h`, I don't think we include changes to all other header files to remove now-unneeded declarations and definitions.

→ #1473